dependency-manager-tests/spm/project.pbxproj.src

```diff
@@ -1,652 +0,0 @@
-// !$*UTF8*$!
```

Comment:
👌 Just double checking - this file is not needed as we use `SPMProject.xcodeproj.src`, true?

Reply:
yes, correct 👍 

```
// Makefile
@rm -rf SPMProject.xcodeproj
@cp -r SPMProject.xcodeproj.src SPMProject.xcodeproj
@sed "s|REMOTE_GIT_BRANCH|${GIT_BRANCH}|g" SPMProject.xcodeproj.src/project.pbxproj > SPMProject.xcodeproj/project.pbxproj
```